jasmine/spec/bowlingSpec.js

```diff
@@ -0,0 +1,619 @@
+describe('BowlingScorecard',function() {
+
+  var bowlingScorecard;
+
+  beforeEach(function() {
+    bowlingScorecard = new BowlingScorecard();
+  });
+
+  describe('Roll Score', function() {
```

Comment:
It is clear that you did a real TDD here with you thinking being guided by tests.